### DISCUSSION TITLE
[ FF7 ] 7th Heaven, Harcore mod

### DISCUSSION THREAD

**Discussion by Strifegarza7777** [julianxhokaxhiu/FFNx#257] [Q&A] [answered]
Fighting Adamantite Turtle enemy near Wutai, first visit to this area. Playing game with Hardcore setting; using the Gameplay Qhimm Catalog mod. I have a party of Cloud, Cid, Vincent. Every time Cid uses his limit break, Hyperdrive (has a chance to kill an opponent instantly), the game crashes. I'm thinking the enemy has some sort of death resistance that's causing the game's logic to fail.

**Comment by julianxhokaxhiu** [comment, marked as answer]
7th Heaven issues have to be reported on the relative officially supported channels. This one is for FFNx direct support. Cheers!

**Comment by Strifegarza7777** [comment]
Much appreciated! I had just followed the prompt that came up in the above example. It said to come here. I'm totally new to this and love this game enough to consider my two cents in fixing it :-) Do you know where I could find these "channels?"

**Reply by julianxhokaxhiu** [reply]
You can either write here http://forums.qhimm.com/index.php?board=37.0 or join the Qhimm Discord https://github.com/julianxhokaxhiu/FFNx#discord and write in the relative 7th heaven channel :)

No worries, I understand it's not your own issue, but this is actually confusing because FFNx prints that error as it can't understand which "mod manager" is including it, so when it crashes it thinks you're running FFNx in vanilla mode. Unfortunately this is not the case. I've also written this when you try to create a new issue but apparently no one reads it.
